conpot/protocols/modbus/modbus_server.py

```diff
@@ -23,34 +27,60 @@ class ModbusServer(modbus.Server):
     def __init__(self, template, template_directory, args, timeout=5):
 
         self.timeout = timeout
+        self.delay = None
+        self.mode = None
         databank = slave_db.SlaveBase(template)
 
         # Constructor: initializes the server settings
-        modbus.Server.__init__(self, databank if databank else modbus.Databank())
+        modbus.Server.__init__(
+            self, databank if databank else modbus.Databank())
 
-        # not sure how this class remember slave configuration across instance creation, i guess there are some
+        # retrieve mode of connection and turnaround delay from the template
+        self._setup(template)
```

Comment:
Generally definition should be before usage. Makes reading the code easier ("Oh, I have seen this!", instead of: "what does this?")